modules/ROOT/pages/configure-ingress-http-resource.adoc

```diff
@@ -0,0 +1,228 @@
+| `{{ .OrganizationID }}`| Replaces with the organization ID of the Mule app deployment.
+| `{{ .EnvironmentID }}`| Replaces with the environment ID of the Mule app deployment.
+| `{{ .UniqueId }}`| Replaces with the unique ID.
+|===
+
+== How HTTP Route-Template Custom Resource Works in Runtime Fabric
+
+This diagram provides an overview of how to configure ingress using an HTTP route-template custom resource in Runtime Fabric:
+
+image::ingress-http-diagram.png[HTTP route-template diagram overview]
+
+The `httproutetemplates` resource applies only to new Mule app deployments created after the template resource is set up. It does not affect existing deployments. If you redeploy a Mule app after creating or updating the `httproutetemplates`, the new template is applied.
+
+The `httproutetemplates` resource enables you to configure routes like these:
+
+* `baseEndpoints` +
+Describes a list of endpoints associated with the routing resources defined in the template. These endpoints are synchronized by Runtime Fabric with the Anypoint control plane. Application developers can select from a list of synchronized base endpoints during deployment and optionally customize the path and subdomain when applicable.
+
+* `resources` +
+Describes a list of Kubernetes routing resource templates, which are created for every matching Mule app ingress endpoint.
+
+To get the CRD of key elements that you can configure for the template, retrieve the Kubernetes specification by running this command:
+
+----
+kubectl get crd httproutetemplates.rtf.mulesoft.com -o yaml
+----
+
+== HTTP Route-Template Examples
+
+Review the following HTTP route templates examples for Kubernetes ingress, OpenShift routes, and Kubernetes Gateway API (HTTPRoute)
+
+
+=== Example for Kubernetes Ingress
+This is an HTTP route-template example resource for Kubernetes ingress:
+
+----
+apiVersion: rtf.mulesoft.com/v1
+kind: HTTPRouteTemplate
+metadata:
+  name: test-template-ingress
+  namespace: rtf
+spec:
+  baseEndpoints:
+    - https://*.example.com
+    - https://example.internal.com
+  resources:
+    - |
+      apiVersion: networking.k8s.io/v1
+      kind: Ingress
+      metadata:
+        name: {{ .ResourceName }}
+        namespace: {{ .Namespace }}
+        labels:
+        annotations:
+      spec:
+       ingressClassName: nginx 
+       rules:
+       - host: {{ .Host }} 
+         http:
+            paths: 
+            - pathType: Prefix
+              path: {{ .Path }}
+              backend: 
+                service:
+                  name: {{ .Service.Name }}
+                  port:
+                    name: {{ .Service.PortName }}
+              tls:
+              - hosts:
+                - {{ .Host }}
+                secretName: tls-ingress
+----
+
+Note the following about the example:
+
+* The `baseEndpoints` placeholder defines the list of hosts or domains used for mapping the public endpoints configured on the Mule app deployment.
```

Comment:
If you use a calloutlist, you don't need to reiterate the code elements - just include the number on the line in the code and match it to the number in the calloutlist. 

See: https://confluence.internal.salesforce.com/display/MTDT/Formatting+with+Asciidoc#FormattingwithAsciidoc-AddCalloutstoCodeBlocks